### ISSUE TITLE
ResNet50 inference model seemed not support FP16

### ISSUE THREAD

**Issue by chenming22** [IntelAI/models#173]
I want to evaluate inference performance using AMX-FP16 on new CPU while default docker or AI tool package not supported AMX-FP16 because of old version of onednn(3.2.0), so building onednn-3.3.0 and intel extensions for tensorflow-2.15.0 from source and using following model: 
   # FP32, FP16 and BFloat32 Pretrained model:
    wget https://zenodo.org/record/2535873/files/resnet50_v1.pb 
howerver, log from onednn showed precision of this model was FP32 not FP16.

**Comment by sramakintel** [comment]
@chenming22 we will work on adding `FP16` to resnet50 model. Thanks for reaching out.

**Comment by sramakintel** [comment]
> @chenming22 we will work on adding `FP16` to resnet50 model. Thanks for reaching out.

@chenming22 we currently have AMP FP16 TF support based on the PR upstream here: https://github.com/tensorflow/tensorflow/pull/62817. Since the script already supports FP16 input, installing `tf-nighty` from pip can provide FP16 support. Can you run the workload again to see if FP16 works as expected?

**Comment by chenming22** [comment]
thanks @sramakintel , i will have a try.

**Comment by chenming22** [comment]
hi, @sramakintel , noticed [above pr](https://github.com/tensorflow/tensorflow/pull/62817) been merged to tf2.16.1, so i install tf2.16.1 from pip and then ran inference.sh in resnet50 [ResNet50 V1.5 inference](https://github.com/intel/models/blob/master/benchmarks/image_recognition/tensorflow/resnet50v1_5/inference/README.md) with precision=fp16, ONEDNN_MAX_CPU_ISA=AVX512_CORE_AMX_FP16 and model: resnet50_v1.pb, but log and metric on emon showed no AMX instructions, any problems in my test?
